### ISSUE TITLE
Error 500 on Gaia launch_job_async

### ISSUE THREAD

**Issue by jvines** [astropy/astroquery#1752]
Hi!

I've been using astroquery on a project of mine and until yesterday the connection to Gaia was working fine, but now I consistently get an error 500 when trying to use it. I tried reproducing this with [this](https://gea.esac.esa.int/archive-help/tutorials/python_cluster/index.html#Gaia-Python-Tutorial:-Cluster-Analysis) example and the same happened. Is something wrong with the Gaia TAP??

Thanks.

**Comment by afkowalski** [comment]
I get the same error!

**Comment by jvines** [comment]
@afkowalski seems to be a problem with Gaia TAP itself, as trying to query with the adql directly on the DR2 archive doesn't work

**Comment by bsipocz** [comment]
cc @jcsegovia

**Comment by jvines** [comment]
Hi, has there been any progress or development with this? I'm still getting Error 500 from Gaia, apparently from a timeout

This is the stack
```python
---------------------------------------------------------------------------
HTTPError                                 Traceback (most recent call last)
<ipython-input-20-69f4fe2ef887> in <module>
----> 1 j = Gaia.launch_job_async(query)

~/anaconda3/lib/python3.7/site-packages/astroquery/utils/tap/core.py in launch_job_async(self, query, name, output_file, output_format, verbose, dump_to_file, background, upload_resource, upload_table_name, autorun)
    441                         job.save_results(verbose)
    442                     else:
--> 443                         job.get_results()
    444                         log.info("Query finished.")
    445         return job

~/anaconda3/lib/python3.7/site-packages/astroquery/utils/tap/model/job.py in get_results(self)
    245         else:
    246             # async: result is in the server once the job is finished
--> 247             self.__load_async_job_results()
    248             return self.results
    249

~/anaconda3/lib/python3.7/site-packages/astroquery/utils/tap/model/job.py in __load_async_job_results(self, debug)
    342             check_launch_response_status(resultsResponse,
    343                                          debug,
--> 344                                          200)
    345         self._phase = phase
    346         if phase == 'ERROR':

~/anaconda3/lib/python3.7/site-packages/astroquery/utils/tap/conn/tapconn.py in check_launch_response_status(self, response, debug, expected_response_status, raise_exception)
    633             errMsg = taputils.get_http_response_error(response)
    634             print(response.status, errMsg)
--> 635             raise requests.exceptions.HTTPError(errMsg)
    636         else:
    637             return isError

HTTPError: Error 500:
```

And this is the query
`select gaia.parallax, gaia.parallax_error, gaia.teff_val, gaia.teff_percentile_lower, gaia.teff_percentile_upper, gaia.radius_val, gaia.radius_percentile_lower, gaia.radius_percentile_upper, gaia.lum_val, gaia.lum_percentile_lower, gaia.lum_percentile_upper from gaiadr2.gaia_source as gaia where gaia.source_id=1741305708895309312`

Thanks!

**Comment by eerovaher** [comment]
I am unable to replicate the bug with astroquery 0.4.4.dev6999. Should this issue be closed?

**Comment by keflavich** [comment]
This may have been an upstream issue.  If it recurs, post a new issue.
